### ISSUE TITLE
Add role button and tabindex to color boxes in Color-Pelette

### ISSUE THREAD

**Issue by ashrion** [dhairyagothi/100_days_100_web_project#7935]
File: public/Color-Pelette/index.html lines 20-24

The color box div elements have onclick handlers but are missing role button and tabindex, making them inaccessible to keyboard users.

GSSoC26 contribution
